python/tests/datasources/test_cdc_restricted.py

```diff
@@ -250,7 +250,7 @@ def testWriteToBq(
     assert mock_bq.call_args_list[3].args[2] == 'by_sex_county_processed'
     assert mock_bq.call_args_list[4].args[2] == 'by_race_county_processed'
     assert mock_bq.call_args_list[5].args[2] == 'by_age_county_processed'
-    assert mock_bq.call_args_list[6].args[2] == 'cdc_restricted_by_race_and_age_state'
+    assert mock_bq.call_args_list[6].args[2] == 'by_race_age_state'
```

Comment:
assuming this is intentional to bring our table naming scheme more into alignment? I've noticed it's inconsistent between data sources / levels

Reply:
This is intentional! the tables that start with `cdc_restricted` are the old ones and will go away completely after we do national calculations on the backend!